azure_functions_worker/dispatcher.py

```diff
@@ -405,6 +406,7 @@ async def _handle__worker_init_request(self, request):
                 if HttpV2Registry.http_v2_enabled():
                     capabilities[constants.HTTP_URI] = \
```

Comment:
nit: import `HTTP_URI` from `.constants `and reference it like `capabilities[HTTP_URI]` instead of `constants.HTTP_URI` to be consistent with `capabilities[REQUIRES_ROUTE_PARAMETERS]`